### PR TITLE
Feature/nexus split

### PR DESCRIPTION
**Pull request by bbakernoaa**
This is a simple merge to the utilities.  It adds a new script, `concatenate_nexus_post_split.py`, that will combine multiple nexus files into a single `NEXUS_Expt_combined.nc` file.

### CODE REVIEW

File: utils/python/concatenate_nexus_post_split.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    from glob import glob
+
+    parser = argparse.ArgumentParser(
+        description="Combine/reassign NEXUS emissions outputs for CMAQ."
+    )
+    parser.add_argument("INPUT", type=str, help="Input file path.")
+    parser.add_argument("OUTPUT", type=str, help="Output file path.")
+    parser.add_argument("--compress", action="store_true", help="Whether to apply compression for the variables.")
+    parser.add_argument("--no-compress", action="store_false", dest="compress")
+    parser.set_defaults(compress=True)
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py:44]: `compress` currently isn't used, do we want to add?

File: utils/python/concatenate_nexus_post_split.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    return 0
+
+
+def parse_args(argv=None):
+    import argparse
+    from pathlib import Path
+    from glob import glob
+
+    parser = argparse.ArgumentParser(
+        description="Combine/reassign NEXUS emissions outputs for CMAQ."
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py]:
```suggestion
        description="Combine outputs from NEXUS split jobs."
```

File: utils/python/make_nexus_output_pretty.py [hunk 3]
```diff
@@ -22,8 +22,12 @@ def get_hemco_dates(time_file='HEMCO_sa_Time.rc'):
             end = dt.datetime.strptime(L, 'END:     %Y-%m-%d %H:00:00\n')
         if L.startswith('TS_EMIS'):
             ts_emis = float(L.split()[1].strip('\n'))
+    print(start, start_base)
```
Comment (zmoon) [inline utils/python/make_nexus_output_pretty.py]: I think could remove the changes here and merge #18 
if you could just first confirm for me that the time-related assertions in #18 sound reasonable

File: utils/python/concatenate_nexus_post_split.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        "ofp": args.OUTPUT,
+        "compress": args.compress,
+    }
+
+
+if __name__ == "__main__":
+    raise SystemExit(main(**parse_args()))
+
+
+
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py]:
```suggestion
```

**Review by zmoon — CHANGES_REQUESTED** [review]

File: utils/python/make_nexus_output_pretty.py (hunk 3 above)
Comment (bbakernoaa) [reply utils/python/make_nexus_output_pretty.py]: good lets do that

File: utils/python/concatenate_nexus_post_split.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+def parse_args(argv=None):
+    import argparse
+    from pathlib import Path
+    from glob import glob
+
+    parser = argparse.ArgumentParser(
+        description="Combine outputs from NEXUS split jobs."
+    )
+    parser.add_argument("INPUT", type=str, help="Input file path.")
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py]:
```suggestion
    parser.add_argument("INPUT", type=str, help="Input directory.")
```

File: utils/python/concatenate_nexus_post_split.py (hunk 1 above)
Comment (zmoon) [reply utils/python/concatenate_nexus_post_split.py:44]: I can add compression for the data vars if you want

File: utils/python/make_nexus_output_pretty.py (hunk 3 above)
Comment (zmoon) [reply utils/python/make_nexus_output_pretty.py]: It's been merged

File: utils/python/concatenate_nexus_post_split.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    files = glob(ifp)
+    files.sort()
+    print(files)
+
+    dsets = []
+    for fi in files:
+        dsets.append(xr.open_dataset(fi))
+    ds = xr.concat(dsets, dim='time')
+    
+    ds.to_netcdf(ofp)
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py]: To use `compress`
```suggestion
    if compress:
        encoding = {vn: {"zlib": True, "complevel": 1} for vn in ds.data_vars}
    else:
        encoding = None
    ds.to_netcdf(ofp, encoding=encoding)
```

File: utils/python/make_nexus_output_pretty.py (hunk 3 above)
Comment (zmoon) [reply utils/python/make_nexus_output_pretty.py]: And I just merged develop here, so now there is no merge conflict

**Comment by bbakernoaa** [comment]
@zmoon The test completed successfully.  Let's merge

**Comment by zmoon** [comment]
Great!

**Pull request merged**